run_dir/design/project_samples.html

```diff
@@ -286,6 +287,23 @@ <h4 class="panel-title"><a data-toggle="collapse" data-parent="#undefined_fields
         </div>
       </form>
       <hr>
+      <!-- Woksets -->
+      <label for="tab_samples_worksets">Worksets:</label>
+      <div id="tab_samples_worksets">
+      {% for workset in worksets %}
+        <div class="btn-group">
+          <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            {{ workset }} <span class="caret"></span>
+          </button>
+          <ul class="dropdown-menu">
+            <li><a href="https://genologics.scilifelab.se/clarity/search?scope=Container&query={{ workset }}">Open in the LIMS</a></li>
+            <li><a href="/workset/{{ workset }}">Open in the GenStat</a></li>
```

Comment:
However you say. I don't really know how to use articles, in my language they don't exist.
@ewels as english expert - agree?